app/models/spree/credit_card_decorator.rb

```diff
@@ -3,6 +3,9 @@
   # Should be able to remove once we reach Spree v2.2.0
   # Commit: https://github.com/spree/spree/commit/5a4d690ebc64b264bf12904a70187e7a8735ef3f
   # See also: https://github.com/spree/spree_gateway/issues/111
+
+  belongs_to :payment_method
```

Comment:
Actually I think we can also remove this line when we upgrade to 2.2.0 - but I'll make it clearer.

On a that note, I was considering adding a check for Spree version when booting the app in dev, which will give a "deprecation warning" type message listing lines like this that could be removed once 2.2 is in - what do you think? Just to avoid a bit of future bloat in the code. Either that or consistent searchable comment we can add to such lines - I guess just the spree version number would be enough.